src/helpers/propPathOr.js

```diff
@@ -20,13 +17,19 @@ function propPathOr(def, keys, target) {
     return def
```

Comment:
I think that `if` condition is not needed anymore since the `reduce` will check that.